### DISCUSSION TITLE
The fileType field could not be obtained

### DISCUSSION THREAD

**Discussion by zmcode** [cdnjs/cdnjs#14176] [General]
https://api.cdnjs.com/libraries?fields=version,fileType

This gets the fileType null

How to obtain ?

**Comment by zmcode** [comment]
![1672649145271](https://user-images.githubusercontent.com/48686959/210209757-dbfe5293-284a-4a86-9b1d-f7bf02b8e240.png)
The document says to pass the fileType field, but returns null

**Reply by MattIPv4** [reply]
👋 Please could you file a bug issue for this in cdnjs/api-server. It looks like we're forcing the fields to be lower-cased when requested, which is causing it to return `filetype: null` rather than `fileType: <type>`.

**Reply by zmcode** [reply]
> 👋 Please could you file a bug issue for this in cdnjs/api-server. It looks like we're forcing the fields to be lower-cased when requested, which is causing it to return `filetype: null` rather than `fileType: <type>`.

https://github.com/cdnjs/api-server/issues/86 I submitted it here

**Comment by MattIPv4** [comment]
👋 This should now be working:

```
curl -sSL api.cdnjs.com/libraries\?fields=fileType | jq -r .results\[0].fileType
js
```
